blocks/library/freeform/freeform-block.js

```diff
@@ -45,6 +45,7 @@ const FREEFORM_CONTROLS = [
 		title: wp.i18n.__( 'Convert to ordered' ),
 	},
 	{
+		showDivider: true,
```

Comment:
Good point. It shows to the left so I suppose the simple solution is to rename it to leftDivider.